config/initializers/audit_instrumentation.rb

```diff
@@ -0,0 +1,32 @@
+# Audit logging from fact importer events sent via ActiveSupport::Notifications
```

Comment:
More docs explaining how `.instrument` stuff would go into this logger would be helpful, I don't expect everyone to know about it right away